tests/bluetooth/tester/Makefile

```diff
@@ -12,12 +12,12 @@ BOARD ?= arduino_101
 
 CONF_FILE ?= default.conf
 
-# UART for Bluetooth
-QEMU_EXTRA_FLAGS = -serial unix:/tmp/bt-server-bredr
-
 # UART for Tester
 QEMU_EXTRA_FLAGS += -serial pipe:/tmp/bt-stack-tester
 
+# UART for Bluetooth
+QEMU_EXTRA_FLAGS = -serial unix:/tmp/bt-server-bredr
+
```

Comment:
So we use the same serial port in qemu for the same purpose.  We utilize the 2nd qemu uart for UART_PIPE in a lot of other qemu tests.  So re-ordering things here so the PIPE in this test is 2nd UART and the BT uart is 3rd.